File: Semestres/11.Febrero-Junio 2022/EstructurasDatosUSB/corte2/Tema2Arboles/Ejemploarbol.py
from Arbol import Arbol

objArbol = Arbol(
    "a",
    Arbol("b",
        Arbol("d",None,None),
        Arbol("e",None,None),
    ),
    Arbol(
        "c",
        Arbol("f",None,None),
        Arbol("g",Arbol("k",None,None),None)
    )
)

print(objArbol.key)

File: Semestres/6.Mayo-Septiembre 2019/FLP/27Mayo/ejemploArbol.cpp

class Arbol{
	
	private:
		Arbol * hizq;
		Arbol * hder;
	
	public:
	Arbol(Arbol * hizq, Arbol * hder){
		
		this->hizq = hizq;
		this->hder = hder;
	}

	~Arbol(){
		
		delete hizq;
		delete hder;
	}	
};



main(){
	
	Arbol objArbol(Arbol(Arbol(),Arbol()),Arbol());
}
